modules/apps/app-builder/app-builder-test/src/testFunctional/macros/AppBuilderAdmin.macro

```diff
@@ -174,6 +200,16 @@ definition {
 			locator1 = "AppBuilder#CHECKBOX_OPTION_LABEL");
 	}
 
+	macro undeployApp {
+		Click(
+			key_itemName = "${itemName}",
+			locator1 = "AppBuilder#VERTICAL_ELLIPSIS_BUTTON");
+
+		MenuItem.clickNoError(menuItem = "Undeploy");
+
+		Click(locator1 = "AppBuilder#UNDEPLOY_APP_BUTTON");
```

Comment:
Hi @nkkwon , could you please add the check as you've added on deployApp macro?

AssertTextEquals(
			locator1 = "AppBuilder#DEPLOY_APP_STATUS",
			value1 = "Undeployed");